.github/workflows/test.yml

```diff
@@ -31,18 +35,9 @@ jobs:
         run: |
           export SBT_OPTS="-Duser.timezone=UTC -Denv.type=test"
           sbt coverage test coverageReport
+
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
         with:
           fail_ci_if_error: true
           token: ${{ secrets.CODECOV_TOKEN }}
-
-      - name: Setup Cache
-        uses: coursier/cache-action@v5
-      - name: Cache resources
-        run: |
-          rm -rf "$HOME/.ivy2/local" || true
-          find $HOME/Library/Caches/Coursier/v1        -name "ivydata-*.properties" -delete || true
-          find $HOME/.ivy2/cache                       -name "ivydata-*.properties" -delete || true
-          find $HOME/.cache/coursier/v1                -name "ivydata-*.properties" -delete || true
-          find $HOME/.sbt                              -name "*.lock"               -delete || true
```

Comment:
removed this cache in favor of the `cache: sbt` from the newer version of `setup-java`